<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Ferramenta de Produção</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="login_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Servidor :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="server_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Usuário :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="user_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Senha :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="password_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>32</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="localhost_check">
     <property name="text">
      <string>Conexão local</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ok_btn">
     <property name="text">
      <string>&amp;Confirmar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="version_text">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ok_btn</tabstop>
  <tabstop>server_input</tabstop>
  <tabstop>user_input</tabstop>
  <tabstop>password_input</tabstop>
  <tabstop>localhost_check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
